### DISCUSSION TITLE
New release of `mdast-util-definitions`?

### DISCUSSION THREAD

**Discussion by wlach** [syntax-tree/unist#71] [General] [answered]
Any chance of a new release of `mdast-util-definitions`? The last release on npm (5.1.0) depends on an old/out of date version of `unist-util-visit` (3.1.0). It appears that this was updated *just after* it was released (https://github.com/syntax-tree/mdast-util-definitions/commit/59fa6ead9ba587e075bbdf0194ea93b0f6d3229f)

**Comment by wlach** [comment, marked as answer]
Might be a question for @wooorm.

**Reply by wooorm** [reply]
released https://github.com/syntax-tree/mdast-util-definitions/releases/tag/5.1.1

**Reply by wlach** [reply]
Thank you!
